features/step_definitions/json_api_average.rb

```diff
@@ -9,9 +9,9 @@
 end
 
 Given(/^a set of ratings for several songs$/) do
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 1 }
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 2 }
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 3 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 1 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 2 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 3 }
```

Comment:
There's probably an easier way to set the initial i value, but this was a quick fix.